File: src/main/java/org/artembogomolova/demo/webapp/security/AuthenticationSuccessHandlerImpl.java
package org.artembogomolova.demo.webapp.security;

import java.io.IOException;
import javax.servlet.ServletException;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
import lombok.extern.slf4j.Slf4j;
import org.springframework.security.core.Authentication;
import org.springframework.security.web.authentication.AuthenticationSuccessHandler;
import org.springframework.stereotype.Service;

@Service
@Slf4j
public class AuthenticationSuccessHandlerImpl implements AuthenticationSuccessHandler {

  @Override
  public void onAuthenticationSuccess(HttpServletRequest httpServletRequest, HttpServletResponse httpServletResponse, Authentication authentication)
      throws IOException, ServletException {
    log.info("authenticated successful! {}", authentication.getName());
  }
}

File: app/src/main/kotlin/org/artembogomolova/demo/webapp/main/security/AuthenticationSuccessHandlerImpl.kt
package org.artembogomolova.demo.webapp.main.security

import org.springframework.security.web.authentication.AuthenticationSuccessHandler
import java.io.IOException
import org.springframework.security.core.Authentication
import org.springframework.stereotype.Service
import javax.servlet.ServletException
import javax.servlet.http.HttpServletRequest
import javax.servlet.http.HttpServletResponse

@Service
class AuthenticationSuccessHandlerImpl : AuthenticationSuccessHandler {
    @Throws(IOException::class, ServletException::class)
    override fun onAuthenticationSuccess(httpServletRequest: HttpServletRequest, httpServletResponse: HttpServletResponse, authentication: Authentication) {
        println("authenticated successful! ${authentication.name}")
    }
}